tests/utils/subnet.go

```diff
@@ -9,41 +9,103 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
 	"time"
 
+	"github.com/ava-labs/avalanchego/api/health"
 	"github.com/ava-labs/avalanchego/api/info"
 	"github.com/ava-labs/avalanchego/genesis"
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
 	wallet "github.com/ava-labs/avalanchego/wallet/subnet/primary"
 	"github.com/ava-labs/subnet-evm/core"
 	"github.com/ava-labs/subnet-evm/plugin/evm"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/go-cmd/cmd"
+	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
 )
 
-// RunTestCMD runs a given test command with the given rpcURI
-// It also waits for the test ping to succeed before running the test command
-func RunTestCMD(testCMD *exec.Cmd, rpcURI string) {
-	log.Info("Sleeping to wait for test ping", "rpcURI", rpcURI)
-	client, err := NewEvmClient(rpcURI, 225, 2)
-	gomega.Expect(err).Should(gomega.BeNil())
-
-	bal, err := client.FetchBalance(context.Background(), common.HexToAddress(""))
-	gomega.Expect(err).Should(gomega.BeNil())
-	gomega.Expect(bal.Cmp(common.Big0)).Should(gomega.Equal(0))
+type SubnetSuite struct {
+	GetBlockchainID func(alias string) string
+}
```

Comment:
This seems pretty jank. Can we define a simple type that stores a map and provides a read/write lock to support this functionality instead?

Reply:
done